.stylelintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "rules": {
+    "number-leading-zero": "never",
+    "color-hex-case": "upper"
+  }
+}
```

Comment:
Am I correct in seeing that we didn't set up the rules from the sass-lint file? It looks like we're losing the current linting we do other than these two rules?

It looks like most of these can be brought over and have parallels?

https://stylelint.io/user-guide/rules/list